doc/getting-started.rst

```diff
@@ -1,90 +1,100 @@
-Getting Started with Haskell and Cabal
-======================================
+Getting Started
+===============
 
-Installing the Haskell toolchain
---------------------------------
+Installing Cabal
+----------------
 
-To install the Haskell toolchain follow the `ghcup instructions
-<https://www.haskell.org/ghcup/>`__.
+The easiest and recommended way to install the ``cabal`` commandline tool 
+on Linux, macOS, FreeBSD or Windows is through `ghcup <https://www.haskell.org/ghcup/>`__. 
+It installs the "Haskell toolchain" which includes Cabal,
+the Haskell compiler `GHC <https://www.haskell.org/ghc/>`__ 
+and optionally other useful Haskell tools.
 
-
-Creating a new application
+Creating a Haskell package
```

Comment:
The word "application" was deliberately chosen so that the newcomers know that this is the section they should read in order to learn how to create applications with cabal. The word "package", although technically ("more") correct, is intimidating: generally, newcomers don't look for creating packages in a broad sense (and as newcomers they don't know the precise technical sense in the case of cabal packages), because a package in a broad sense is a piece of reusable functionality, which is not what newcomers are after, most of the time.

Reply:
I feel strongly about making a distinction between a package and a project. A package is the unit of source code distribution. If you are a deploying applications, you are not distributing source code and you have a different set of problem from those who do.
I am on mobile and I dont remember the content of this section but "application" ~ project.

Reply:
Indeed. "application" seems to be target outcome of this mini-guide, which also fits the "myapp" package name.